Test/Unit/Helper/Webhook/AuthorisationWebhookHandlerTest.php

```diff
@@ -1,130 +1,363 @@
+        );
+
+        // Perform assertions on the result and expected behavior
+        $this->assertInstanceOf(Order::class, $result);
+    }
+
+    public function testHandleManualCapture()
+    {
+        // Set up expectations for handleManualCapture private method
+        $requireFraudManualReview = true; // Assuming this condition is met
+        $this->orderHelperMock->expects($this->never()) // Since the condition is true
+        ->method('setPrePaymentAuthorized');
+
+        $this->caseManagementMock->expects($this->once())
+            ->method('markCaseAsPendingReview')
+            ->with($this->orderMock, $this->notificationMock->getPspreference(), false);
+
+        // Create an instance of AuthorisationWebhookHandler
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            $this->orderHelperMock,
+            $this->caseManagementMock,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
+        );
+
+        // Use Reflection to access the private method
+        $handleManualCaptureMethod = $this->getPrivateMethod(
+            AuthorisationWebhookHandler::class,
+            'handleManualCapture'
+        );
+
+        // Call the private method directly and provide required parameters
+        $result = $handleManualCaptureMethod->invokeArgs(
+            $webhookHandler,
+            [$this->orderMock, $this->notificationMock, true]
+        );
+
+        // Perform assertions on the result and expected behavior
+        $this->assertInstanceOf(Order::class, $result);
+    }
+
+    public function testCanCancelPayByLinkOrder()
+    {
+        // Create mocks for the required dependencies
+        $paymentMock = $this->getMockBuilder(Order\Payment::class)
+            ->disableOriginalConstructor()
+            ->getMock();
+
+        // Configure the payment mock to return the payByLinkFailureCount
+        $payByLinkFailureCount = 3; // Assuming the failure count is 3
+
+        // Use willReturn() to mock chained method calls
+        $paymentMock->expects($this->once())
+            ->method('getAdditionalInformation')
+            ->with('payByLinkFailureCount')
+            ->willReturn($payByLinkFailureCount);
+
+        // Configure the order mock to return the payment mock
+        $this->orderMock->method('getPayment')->willReturn($paymentMock);
+
+        $paymentMock->expects($this->once())
+            ->method('setAdditionalInformation')
+            ->with('payByLinkFailureCount', $payByLinkFailureCount + 1);
+
+        // Create an instance of AuthorisationWebhookHandler
+        $webhookHandler = $this->createAuthorisationWebhookHandler(
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null,
+            null
```

Comment:
You can skip sending values for optional arguments. They are already null by default.